coss/settings.py

```diff
@@ -145,6 +146,15 @@
     }
 }
 
+# S3 storage
+AWS_STORAGE_BACKET_NAME = config('AWS_STORAGE_BACKET_NAME', default='')
+AWS_ACCESS_KEY_ID = config('AWS_ACCESS_KEY_ID', default='')
+AWS_SECRET_ACCESS_KEY = config('AWS_SECRET_ACCESS_KEY', default='')
+AWS_S3_CUSTOM_DOMAIN = '{0}'.format(AWS_STORAGE_BACKET_NAME)
```

Comment:
I suggest we do something like this:

```
AWS_S3_CUSTOM_DOMAIN = '%s.s3.amazonaws.com'.format(AWS_STORAGE_BUCKET_NAME)
```

or leave it empty and configure it separately as @flamingspaz says.